test/catalog.tests.bom

```diff
@@ -22,6 +22,11 @@ brooklyn.catalog:
         launch.command: |
           while true ; do sleep 3600; done & echo $! > ${PID_FILE}
         checkRunning.command: ""
+        brooklyn.config:
+          pre.install.command:
+            sudo yum install -y yum-plugin-fastestmirror
+            sudo yum update -y
```

Comment:
I'm hesitant about this for two reasons:
1. Is this the only way it works? Does it fail if we haven't done `yum update`, in which case should we do it in the cassandra-node entity?
2. It feels like this kind of thing should be part of a "hardening script" that could optionally be run, but should not be done/needed in general (i.e. the entity installs/updates only what it needs, unless explicitly configured otherwise).

Given it's in the test-code, we can ignore (2) but (1) is a concern.